proxy/src/registry.rs

```diff
@@ -34,12 +34,24 @@ pub struct Metadata {
 
 /// Possible messages a [`Transaction`] can carry.
 pub enum Message {
-    /// Issue a new project registration with (domain, name).
+    /// Issue a new project registration with a given name under a given org.
     ProjectRegistration {
-        /// Domain part of the project id.
-        domain: String32,
-        /// Actual project name, unique for domain.
-        name: String32,
+        /// Actual project name, unique for org.
+        project_name: ProjectName,
+        /// The Org in which to register the project.
+        org_id: OrgId,
+    },
+
+    /// Issue a new org registration with a given id.
+    OrgRegistration {
```

Comment:
If it's a single field we can use: `OrgRegistration(OrgId)`.

Reply:
Good point!

Reply:
✅